File: 1791-richest-customer-wealth/richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function (accounts) {
    let max = 0;
    for (let i = 0; i < accounts.length; i++) {
        let tm = 0;
        for (let j = 0; j < accounts[0].length; j++) {
            tm += accounts[i][j];
        }
        max = Math.max(tm, max);
    }
    return max;
};

File: 1791-richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = 0;
        for(int i = 0; i < accounts.length; i++){
            int tm = 0;
            for(int j = 0; j < accounts[0].length; j++){
                tm += accounts[i][j];
            }
            max  = Math.max(tm,max);
        }
        return max;
    }
}